dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/GitHubHelper.java

```diff
@@ -143,34 +143,27 @@ public static LicenseInformation getLicenseInformation(GitHub gitHub, String rep
 
     /**
      * Based on GitHub username, find the corresponding user
+     *
      * @param tokenDAO
      * @param userDAO
      * @param username GitHub username
-     * @param allowFail If true, throw a failure if user cannot be found
      * @return user with given GitHub username
      */
-    public static User findUserByGitHubUsername(TokenDAO tokenDAO, UserDAO userDAO, String username, boolean allowFail) {
+    public static Optional<User> findUserByGitHubUsername(TokenDAO tokenDAO, UserDAO userDAO, String username) {
```

Comment:
Regarding the removal of the `allowFail` parameter, this method was always invoked with its value set to `false`, so I just removed it make the method easier to read.